divi/src/masternode-payments.cpp

```diff
@@ -469,22 +465,25 @@ bool CMasternodePayments::AddWinningMasternode(const CMasternodePaymentWinner& w
         return false;
     }
 
+    CMasternodeBlockPayees* payees;
     {
         LOCK2(cs_mapMasternodeBlocks, cs_mapMasternodePayeeVotes);
 
-        if (mapMasternodePayeeVotes.count(winnerIn.GetHash())) {
+        auto* winner = GetPaymentWinnerForHash(winnerIn.GetHash());
+        if (winner == nullptr)
```

Comment:
Should be != nullptr, since it's a shortcut to avoid re-adding a known winner

Reply:
Indeed, good catch!  I've fixed it.